src/spec/sync/product-sync.spec.coffee

```diff
@@ -226,6 +226,120 @@ describe 'ProductSync', ->
         version: oldProduct.version
       expect(update).toEqual expected_update
 
+    it 'should create `changePrice` action if new price has id', ->
```

Comment:
If we always match by scope than we do not need a test for price IDs anymore, right?

Reply:
The first check is still the ID. This way, a scope can also be "updated" if the ID stays the same